app/src/state/editor/io.ts

```diff
@@ -66,7 +66,7 @@ export const closeDocument = createAsyncActionWithReducer<EditorState>(
 );
 
 export const setSources = createActionWithReducer<EditorState, Record<string, Source>>(
-  'editor/setState',
+  'editor/setSources',
```

Comment:
:D 

Is there some way to auto-generate those through some cool js-introspection so this doesn't happen?

Reply:
hmm... I cant think of anything at the moment. That redux-toolkits createAsyncThunk cant do this is a hint that this is not possible?